index.html

```diff
@@ -421,14 +421,15 @@ <h2>I'm always interested in hearing about new projects, so if you'd like to cha
                     </div>
                     <div class="form-input">
                         <label for="email"></label>
-                        <input type="email" id="email" name="email" placeholder="Email address" required>
+                        <input type="email" id="emailfield" name="email" placeholder="Email address" required>
+                        <div id="error"></div>
                         <div class="line-form"></div>
                     </div>
                     <div class="form-input">
                         <label for="comment"></label>
                         <textarea name="message" maxlength="500" required id="comment"  cols="30" rows="5" placeholder="Write me something..."></textarea>
                     </div>
```

Comment:
- [ ] It would be better if we can submit the form with email fields all in lowercase, but there is an error in the browser console preventing form submission. Also, we are unable to display the error message when the email field contains an uppercase letter. Here is a screenshot of the problem
![Screenshot from 2023-05-18 10-04-05](https://github.com/aradradev/abdulsportfolio/assets/46388113/2a1119a6-2d55-4045-beb5-216d5d78faf0)